src/domain/estimations/entities/estimation.entity.ts

```diff
@@ -0,0 +1,3 @@
+export class Estimation {
+  safeTxGas: string;
+}
```

Comment:
OOC: why not use an `interface` (to be more aligned with the other entities)?

Reply:
I agree, changed in [cc1b105](https://github.com/5afe/safe-client-gateway-nest/pull/269/commits/cc1b10550ff69e6516e4a0956cc3820a7d6f5c43)